app/views/dashboard/_status_widget.html.erb

```diff
@@ -23,7 +23,7 @@
   %>
 
   <%= searchable_links _('Out of sync hosts'),
```

Comment:
They appear to be for different searches though, this request is just for the out of sync one.